index.js

```diff
@@ -82,6 +82,11 @@ const hasPromiseAPI = x =>
 
 is.promise = x => is.nativePromise(x) || hasPromiseAPI(x);
 
+is.generator = x => is.iterable(x) && is.function(x.next) && is.function(x.throw);
+
+// Change to use `isObjectOfType` once Node 4.x.x LTS is dropped
+is.generatorFunction = x => x.constructor.name === 'GeneratorFunction';
```

Comment:
My first comment in https://github.com/sindresorhus/is/pull/4#discussion_r140665751 still applies.

Reply:
Yeah, I'm sorry about that. Yesterday was not just my day when it came to coding 🙊